<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskManager</class>
 <widget class="QWidget" name="TaskManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskManager</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>82</y>
     <width>601</width>
     <height>351</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="sys_info">
    <attribute name="title">
     <string>系统</string>
    </attribute>
    <widget class="QGroupBox" name="pc_group">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>491</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>此电脑</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>主机名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>启动时间：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>已运行时间：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>版本号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pc_name">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_boot_time">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_run_time">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pc_version">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="cpu_group">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>491</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>CPU</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>型号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>主频：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cpu_name">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cpu_hz">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="pro_info">
    <attribute name="title">
     <string>进程</string>
    </attribute>
    <widget class="QListWidget" name="list_process">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>511</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_kill">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>280</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>结束进程</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_pid">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入进程pid</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_search">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>280</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="cpumem_info">
    <attribute name="title">
     <string>性能</string>
    </attribute>
    <widget class="QGroupBox" name="group_cpu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>521</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>CPU使用率</string>
     </property>
     <widget class="QLabel" name="label_cpurate">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>当前使用率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>当前时间：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_time">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="group_mem">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>521</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>内存使用情况</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>已用：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>可用：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>内存总大小</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_mem_used">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_mem_free">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_mem_total">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="group_swap">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>521</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>交换分区使用情况</string>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>已用：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>可用：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>内存总大小</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_swap_used">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_swap_free">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_swap_total">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>81</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="my_info">
    <attribute name="title">
     <string>关于</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>281</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>获取帮助请联系630212894@qq.com</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_off">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>450</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>关机</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>13</y>
     <width>90</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>当前时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cpurate_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>90</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>内存已用：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>90</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>内存可用：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mem_used_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mem_free_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>99</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CPU使用率：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
